src/app/app.html

```diff
@@ -6,7 +6,7 @@
   <ion-content>
     <div class="menuContainer">
       <div class="menuPresentationProfil">
-        <img src="https://marketplace.canva.com/MAB2si1vbe4/1/thumbnail/canva-business-people-design-person-icon--MAB2si1vbe4.png" alt="" class="photoMenu">
+        <img src="{{menuContact?.gravatar}}?s=200&d=mm" alt="" class="photoMenu">
```

Comment:
MenuContact should be calledMenuUser to be logical since it's the user and not a contact